.github/workflows/e2e-tests.yml

```diff
@@ -35,14 +35,14 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        kubernetes: [v1.18.6,v1.12.0]
+        kubernetes: [v1.19.2,v1.18.6,v1.13.1]
     steps:
       - name: Checkout
         uses: actions/checkout@v2
       - name: Setup Minikube-Kubernetes
         uses: manusa/actions-setup-minikube@v2.0.1
         with:
-          minikube version: v1.12.1
+          minikube version: v1.13.1
```

Comment:
Why?

Reply:
I wanted to add integration tests for Kubernetes `1.19.x`

Reply:
Sorry, comment in wrong place. I'm talking about he upgrade to v1.13.1 if the minimum k8s version.

Reply:
I checked and seems like Kubernetes version 1.12.1 is less than the oldest supported version: v1.13.0 for minikube v1.13.1

Reply:
If tests pass we should keep the older version